Vagrantfile

```diff
@@ -1,5 +1,6 @@
 Vagrant.configure(2) do |config|
   config.vm.box = "ubuntu/xenial64"
+  config.vm.box_version = "20190909.1.0"
```

Comment:
@trung can we please use >= per https://www.vagrantup.com/docs/boxes/versioning.html#version-constraints